src/main/kotlin/no/elhub/auth/features/requests/create/command/RequestCommand.kt

```diff
@@ -0,0 +1,18 @@
+package no.elhub.auth.features.requests.create.command
+
+import no.elhub.auth.features.common.PartyIdentifier
+import no.elhub.auth.features.requests.AuthorizationRequest
+
+sealed interface RequestMetaMarker {
+    fun toMetaAttributes(): Map<String, String>
+}
+
+sealed class RequestCommand(
+    val requestedFrom: PartyIdentifier,
+    val requestedBy: PartyIdentifier,
+    val meta: RequestMetaMarker,
```

Comment:
:art: The Command should contain the data we need to execute the handler, I think these commands are too generic. We have a generic endpoint, but I reckon we should split it up to separate commands to best handle validation for them 🙂